src/install.ts

```diff
@@ -146,4 +156,11 @@ export default async function install(): Promise<void> {
         cwd: process.cwd(),
         respectNpm5: true
     });
+
+    console.log('test script:');
```

Comment:
do we need console.log here?

Reply:
yes, it informs user how to run qava tests depends on module system selected

Reply:
Could it be better with winston logger?

Reply:
I don't think so, it's not a log, it's console message for the user

Reply:
`colsole.log` looks like debugging snippet occasionly left by a developer, whereas `console.info` looks rather neutral, though both provide the same functionality. It's just my opinion, not blocking the pull request.

Reply:
anyway I think it's should not be a blocker for current PR